src/server/ocpp/json/JsonRestWSConnection.ts

```diff
@@ -83,7 +83,7 @@ export default class JsonRestWSConnection extends WSConnection {
         source: this.getChargingStationID(),
         module: MODULE_NAME,
         method: 'handleRequest',
-        message: 'Charging Station is not connected to the backend',
+        message: `Charging Station '${this.getChargingStationID()}' is not connected to the backend`,
```

Comment:
No need to add ${this.getChargingStationID()} as it is already provided in the 'source'.